Add support for Windows OVS Containerisation in Install-OVS script (#5055)
antrea-io/antrea: docs/windows.md

Reviewer: In the paragraph from L24 to L29, we can add details about this change that we support running ovs inside container.

antrea-agent and kube-proxy run as processes on host and are managed by
management Pods. It is recommended to run OVS daemons as Windows services.

> We also support running OVS processes inside container.

Reviewer 2: Could you adjust lines to wrap at column 80?

> We also support running ovs daemons as processes inside container.

Reviewer 3: ```suggestion
We also support running OVS processes inside container.
```

.\Install-OVS.ps1 -ImportCertificate $false -Local -LocalFile <PathToOVSPackage> # Production
```

If you want to containerise OVS for containerd runtime , use the ContaineriseOVS parameter as true.

```powershell
.\Install-OVS.ps1 -ContaineriseOVS $true 
```

> Verify the OVS services are installed.

Reviewer: we can specify here that this is not applicable if you Containerise OVS.

.\Install-OVS.ps1 -ImportCertificate $false -Local -LocalFile <PathToOVSPackage> # Production
```

> If you want to containerise OVS for containerd runtime , use the ContaineriseOVS parameter as true.

Reviewer 4: containerd runtime,

> If you want to containerise OVS for containerd runtime , use the ContaineriseOVS parameter as true.

Reviewer 3: ```suggestion
If you want to containerise OVS for containerd runtime , use the `ContaineriseOVS` parameter as true.
```

```powershell
.\Install-OVS.ps1 -ContaineriseOVS $true 
```

> Verify the OVS services are installed.(not applicale if you containerise OVS)

Reviewer 4: Add space after the period.

Reviewer 3: Typo for "applicale". BTW, maybe this line can be removed?

Author: @wenyingd You mean we should remove the entire section where we are verifying that ovs-services are installed or not using get-service command?

Reviewer 3: No, I mean it looks wierd with the changes. The verification is for OVS services, but the added command actually does not add OVS services. Is it better to add the example command with `-ContaineriseOVS $true` after the verification part?

Author: Okay , understood

get-service ovs-vswitchd
```

> If you want to containerise OVS for containerd runtime, you don't need to install

Reviewer 3: ```suggestion
If you want to containerise OVS for containerd runtime, OVS userspace processes is not required on host and hence you can use the `InstallUserspace` parameter 
as false.
```

> If you want to containerise OVS for containerd runtime, OVS userspace processes is

Reviewer 2: "containerize" is used more?

Reviewer 2: is -> are

> not required on host and hence you can use the `InstallUserspace` parameter

Reviewer 2: "not required" -> "not run"?

on the host

> as false.

Reviewer 2: set the `InstallUserspace` parameter to false
